CONVENTIONS.md

```diff
@@ -222,6 +225,10 @@ spec:
       - operator: Exists
 ```
 
+Tolerating all taints should be reserved for DaemonSets and static
```

Comment:
Which static pods are you referring to?

Reply:
Is there more than one type?

Reply:
I meant which specific static pods in OpenShift. IIUC, static pods should be tolerating all `NoExecute` taints not `NoSchedule` as their manifests are directly present on the node, there is no need for them to go through scheduling cycle.

Reply:
There isn't an option to tolerate all taints of a certain effect, just "All" generally.